spec.md

```diff
@@ -495,7 +504,26 @@ schema being used.
                   "type": "string"
                 }
               }
-            }
+            },
+            "response": {
+              "$schema": "http://json-schema.org/draft-04/schema#",
+              "type": "object",
+              "properties": {
+                "credentials" : {
```

Comment:
It would be extremely useful to have a way to indicate that a particular part of the bind response is secret vs. non-secret.  In k8s, we would be able to use this to differentiate storage for these keys into secret and configmap, respectively.

Reply:
json schema allows you to add your own custom properties. We could add a field like `{... "secret":true, ...}` that the platform will have to be coded to look for and do this. I would want an implementer to try it and then feed that functionality back into the spec vs starting with it. It would not break forwards compatibility if it was added later.